### ISSUE TITLE
Midi Input 

### ISSUE THREAD

**Issue by Morgen82** [matkatmusic/ChordieApp_IssueTracker#16]
The last item in the list (bottom) is not selectable.

Win 7-64 Pro

**Comment by matkatmusic** [comment]
You removed your screenshot.   If you can update this with more information i'll know what to look for.   Also, windows 7 is not supported.

**Comment by matkatmusic** [comment]
Feel free to add more info and reopen the issue.

**Comment by Morgen82** [comment]
I don't know who you are but, Chordieapp@gmail.com asked me to beta test specifically because I am on Windows 7.  I put  Win 7-64 Pro on my issues reports to remind everyone of this with regard to this issue report.

Chordieapp@gmail wrote:
_Would you mind trying the beta and seeing if it runs on it?  I don't have access to windows 7 so I can't test._  

On Sun, Jun 14, 2020, 2:57 PM Morgen 82 <morgen82@gmail.com> wrote:

    Hi,

         Ditching support for Windows 7 is a mistake.

   **https://www.statista.com/chart/8552/windows-7-is-still-the-desktop-os-king**/

    Morgen Donner

**Comment by Morgen82** [comment]
![Midi Input](https://user-images.githubusercontent.com/66924750/84607376-c8137300-ae61-11ea-980b-d22c00712311.PNG)

**Comment by Morgen82** [comment]
You shouldn't need a picture to figure this out.

**Comment by matkatmusic** [comment]
This was the screenshot you shared, which you then deleted without providing further context: 
![Screen Shot 2020-06-14 at 6 31 06 PM](https://user-images.githubusercontent.com/2929847/84609812-4295c000-ae6d-11ea-92f0-90ef584f3e98.png)

Removing information from your posts does not help me understand the issue you are experiencing. 
```
First of all, I don't know who you are but, Chordieapp@gmail.com asked me to beta test 
specifically because I am on Windows 7.
```
I am the developer of ChordieApp.  https://www.gumroad.com/matkatmusic 

`You shouldn't need a picture to figure this out.`
Sentences like this are not helpful toward the cause of fixing issues in the app.  
You did not provide all of the information I requested in the Readme.md when you created the issue.
I'll remind you of what I need: 
```
If you create an issue: Please provide:

- Operating System version
- Steps to reproduce the issue.
- expected behavior
- behavior actually experienced.
- crash log if you happen to have one.
If I can't reproduce it, i can't fix it.
```
if you want to help me with the issue you're experiencing, you can do the following:
- Please provide a video or screenshots depicting the behavior you are experiencing.
- Please provide a description of what you expect to happen.

If you would prefer to not continue beta testing, that is fine too.

**Comment by Morgen82** [comment]
the screen clip with Fretboard Prefences was a mistake.  That's why I deleted it.
The other clip above that with the Midi Input selected is the correct clip for this issue. (repeated below)

I selected the bottom item MIDIIN2(MIDISPORT 2x2 Anniv) and expect it to put a check mark next to that item.  Instead it's putting a check mark next to the item above it. 
No matter which item in the list I select, it checks the item above. 
It's not just a cosmetic issue. I tested it by selecting the second from the bottom - MIDISPORT 2x2 Anniv - and my keyboard controller stops working because it's connecting to AND putting a check mark next to - AkaiPro Internal MIDI. (which isn't connected to anything)
![Midi Input](https://user-images.githubusercontent.com/66924750/84614036-bdfd6e80-ae79-11ea-994b-78b288af16b4.PNG)
